index.html

```diff
@@ -9,7 +9,7 @@
     <meta name="description" content="Avocado - Automated Testing Framework">
     <meta name="author" content="">
 
-    <title>Avocado - Testing made easier</title>
+    <title>Avocado Testing Framework</title>
```

Comment:
OK, I don't have any strong opinion, but the window title `Avocado Framework` looks indeed better. Merging...